docs/source/setup/index.rst

```diff
@@ -1,16 +1,19 @@
 Getting started with SEPAL
 ==========================
 
-Introduction
+Set up an account and learn how to access the SEPAL interface
 ------------
```

Comment:
```suggestion
---------------------------------------------------------------
```

this is something you cannot see but when the documentation is built I see some parsing errors. 
Lines under the title need to be the same length